src/SUMMARY.md

```diff
@@ -1,5 +1,15 @@
 # Summary
 
-- [Onion client](./onion_client.md)
-- [Messenger packets](./messenger_packets.md)
-- [Conference packets](./conference_packets.md)
+[Introduction](intro.md)
+
+- [Cryptography](crypto/crypto.md)
+- [Network](network/network.md)
+- [DHT](dht/dht.md)
+- [TCP Relay](relay/relay.md)
+- [Onion](onion/onion.md)
+    - [Onion client](./onion/client.md)
+- [Transport Protocol](transport/transport.md)
+- [Messenger](messenger/messenger.md)
+    - [Messenger packets](./messenger/packets.md)
+- [Conferences](conferences/conferences.md)
+    - [Conference packets](./conference/packets.md)
```

Comment:
Shouldn't conferences be a chapter of Messenger?

Reply:
They are a standalone thing, AFAIK.

Reply:
How about adding a chapter for File Transfer?

Reply:
Probably a good idea.

Reply:
> They are a standalone thing, AFAIK.

They are but they are at the same level as Messenger packets. See https://toktok.ltd/spec#messenger